test/integration/targets/vmware_vsan_health_info/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+# Test code for the vmware_vsan_health_info module.
+# Copyright: (c) 2019, OVH SAS
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- when: vcsim is not defined
+  block:
+    - name: Gather health info from a vSAN's cluster
+      vmware_vsan_health_info:
+        hostname: "{{ vcenter_hostname }}"
+        username: "{{ vcenter_username }}"
+        password: "{{ vcenter_password }}"
+        cluster_name: "{{ vcenter_cluster_name }}"
+        validate_certs: "{{ vcenter_validate_certs }}"
```

Comment:
Please just use `False` here, we don't properly expose any variable yet to toggle this parameter in our tests:
```yaml
    validate_certs: False
```

Reply:
Done